source/code-generation/query-converter/manage-queries.txt

```diff
@@ -0,0 +1,41 @@
+.. _rm-manage-queries:
+
+==============
+Manage Queries
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator provides various methods for managing your 
+queries while converting and testing.
```

Comment:
(suggestion) 
Couple questions or thoughts. 

Are these available only during conversion and testing? 

These are more like query management options, rather than various methods, no? (Distinction I'm making might be subjective. But I think if you take advantage of "methods" the idea is that you should be able to choose any of them and arrive at the same outcome. Where as this is more a description of actions you can take.)